.github/workflows/daily-deploy-production.yml

```diff
@@ -245,6 +245,7 @@ jobs:
         env_variable_name: SLACK_BOT_TOKEN
         
     - name: Get va-vsp-bot token
+      if: ${{ env.DSVA_SCHEDULE_ENABLED == 'true' }}
```

Comment:
Looks like I missed a statement. This will make sure we have a nice green check mark when `DVSA_SCHEDULE_ENABLED` is `false` :)